--- a/PKGBUILD
+++ b/PKGBUILD
@@ -19,7 +19,7 @@
 b2sums=('00204ed5c71f5601e2dfa33dd38b32abcd66d9f9257c8a0d3a1c678529c5a94f2cb31a623001a6ffd950a508bd583ad9dabaefb03de112c97c362a32f7264df8')
 
 build() {
-  cmake -S libdeflate -B build -G Ninja \
+  cmake -S libdeflate -B build -G Ninja -D CMAKE_INSTALL_LIBDIR=lib \
     -DCMAKE_BUILD_TYPE=None \
     -DCMAKE_INSTALL_PREFIX=/usr \
     -DLIBDEFLATE_BUILD_STATIC_LIB=OFF \
