--- a/PKGBUILD
+++ b/PKGBUILD
@@ -9,22 +9,20 @@
 url='https://jpeg.org/jpegxl/'
 license=('BSD-3-Clause')
 makedepends=(
-    'add-determinism'
-    'asciidoc'
-    'brotli'
-    'cmake'
-    'doxygen'
-    'giflib'
-    'git'
-    'gperftools'
-    'graphviz'
-    'gtest'
-    'highway'
-    'java-environment'
-    'libjpeg-turbo'
-    'libpng'
-    'python'
-    'xdg-utils')
+ 'asciidoc'
+ 'brotli'
+ 'cmake'
+ 'doxygen'
+ 'giflib'
+ 'git'
+ 'gperftools'
+ 'graphviz'
+ 'gtest'
+ 'highway'
+ 'libjpeg-turbo'
+ 'libpng'
+ 'python'
+ 'xdg-utils')
 source=("git+https://github.com/libjxl/libjxl.git#tag=v${pkgver}"
         'git+https://github.com/mm2/Little-CMS.git'
         'git+https://github.com/webmproject/sjpeg.git'
