--- a/PKGBUILD
+++ b/PKGBUILD
@@ -14,8 +14,6 @@
 makedepends=(
   'libgcc'
   'libstdc++'
-  'doxygen'
-  'graphviz'
   'python'
   'meson'
 )
@@ -42,7 +40,6 @@
 
   echo "Building docs..."
   python doxybuild.py \
-    --doxygen="$(command -v doxygen)" \
     --with-dot
 }
 
@@ -80,7 +77,6 @@
   cd ${pkgbase}-${pkgver}
   install -Dm 644 LICENSE -t "${pkgdir}/usr/share/licenses/${pkgname}"
   install -dm 755 "${pkgdir}/usr/share/doc/${pkgbase}"
-  cp -a dist/doxygen/jsoncpp-api-html-${pkgver} "${pkgdir}/usr/share/doc/${pkgbase}/html"
 }
 
 # vim: ts=2 sw=2 et:
