--- a/PKGBUILD
+++ b/PKGBUILD
@@ -9,7 +9,7 @@
   glib2-docs
 )
 pkgver=2.88.3
-pkgrel=1
+pkgrel=5
 pkgdesc="Low level core library"
 url="https://gitlab.gnome.org/GNOME/glib"
 license=(LGPL-2.1-or-later)
@@ -25,7 +25,7 @@
 )
 makedepends=(
   dbus
-  dconf
+  gcc
   gettext
   gi-docgen
   git
@@ -35,13 +35,8 @@
   python
   python-docutils
   python-packaging
-  shared-mime-info
   util-linux
 )
-checkdepends=(
-  desktop-file-utils
-  glib2
-)
 source=(
   "git+https://gitlab.gnome.org/GNOME/glib.git?signed#tag=$pkgver"
   "git+https://gitlab.gnome.org/GNOME/gvdb.git"
@@ -106,10 +101,6 @@
   meson compile -C build
 }
 
-check() {
-  meson test -C build --no-suite flaky --no-suite slow --print-errorlogs
-}
-
 _pick() {
   local p="$1" f d; shift
   for f; do
@@ -178,6 +169,8 @@
   mv devel/* "$pkgdir"
 }
 
+# vim:set sw=2 sts=-1 et:
+
 package_glib2-docs() {
   pkgdesc+=" - documentation"
   depends=()
@@ -186,5 +179,3 @@
   mv docs/* "$pkgdir"
   install -Dt "$pkgdir/usr/share/licenses/$pkgname" -m644 glib/docs/reference/COPYING
 }
-
-# vim:set sw=2 sts=-1 et:
