--- a/PKGBUILD
+++ b/PKGBUILD
@@ -9,7 +9,7 @@
 arch=('x86_64')
 license=('LGPL-2.1-only')
 depends=('alsa-lib' 'glibc')
-makedepends=('jack' 'libpulse' 'sdl2-compat')
+makedepends=()
 optdepends=(
   'sdl2-compat: for sdl audio support'
   'jack: for jack audio support'
@@ -27,7 +27,7 @@
   ./configure \
     --prefix=/usr \
     --enable-int-quality \
-    --with-audio="alsa oss sdl jack pulse"
+    --with-audio="alsa oss"
   # https://bugzilla.gnome.org/show_bug.cgi?id=655517
   sed -i -e 's/ -shared / -Wl,-O1,--as-needed\0/g' libtool
   make
