--- a/PKGBUILD
+++ b/PKGBUILD
@@ -4,7 +4,6 @@
 pkgbase=pango
 pkgname=(
   pango
-  pango-docs
 )
 pkgver=1.58.2
 pkgrel=1
@@ -28,7 +27,7 @@
   libxrender
 )
 makedepends=(
-  gi-docgen
+  gcc
   git
   glib2-devel
   gobject-introspection
@@ -41,11 +40,14 @@
 
 prepare() {
   cd pango
+
+  # clang 23 escalates -Wunused-but-set-global via this flag (glib macros)
+  sed -i "/'-Werror=unused-but-set-variable',/d" meson.build
 }
 
 build() {
   local meson_options=(
-    -D documentation=true
+    -D documentation=false
     -D man-pages=true
     -D sysprof=enabled
   )
@@ -62,15 +64,6 @@
 
   meson install -C build --destdir "$pkgdir"
 
-  mkdir -p doc/usr/share
-  mv {"$pkgdir",doc}/usr/share/doc
-}
-
-package_pango-docs() {
-  pkgdesc+=" (documentation)"
-  depends=()
-
-  mv doc/* "$pkgdir"
 }
 
 # vim:set sw=2 sts=-1 et:
