--- a/PKGBUILD
+++ b/PKGBUILD
@@ -10,11 +10,8 @@
 conflicts=('libv4l')
 backup=(etc/rc_maps.cfg)
 license=('LGPL')
-depends=('hicolor-icon-theme' 'libgcc' 'libbpf' 'libjpeg-turbo'  'systemd-libs' 'json-c')
-makedepends=('qt6-base' 'qt6-5compat' 'alsa-lib' 'meson' 'clang' 'doxygen')
-optdepends=('qt6-base: for qv4l2 and qvidcap'
-            'qt6-5compat: for qv4l2'
-            'alsa-lib: for qv4l2')
+depends=('hicolor-icon-theme' 'libgcc' 'libjpeg-turbo'  'systemd-libs' 'json-c')
+makedepends=('meson' 'clang')
 source=(https://linuxtv.org/downloads/v4l-utils/${pkgname}-${pkgver}.tar.xz{,.asc})
 sha256sums=('6828828a17775526eb93fb258a9294d1d1073d633c344dd71ecd4e7a1ffb7dfc'
             'SKIP')
@@ -27,7 +24,7 @@
 }
 
 build() {
-  arch-meson -Dgconv=disabled "${pkgname}-${pkgver}" build
+  arch-meson -Dgconv=disabled -Dqv4l2=disabled -Dqvidcap=disabled -Ddoxygen-doc=disabled -Dbpf=disabled "${pkgname}-${pkgver}" build
   meson compile -C build
 }
 
