--- a/PKGBUILD
+++ b/PKGBUILD
@@ -3,7 +3,7 @@
 # Contributor: speps <speps at aur dot archlinux dot org>
 
 pkgbase=serd
-pkgname=(serd serd-docs)
+pkgname=(serd)
 pkgver=0.32.10
 pkgrel=1
 pkgdesc="Lightweight C library for RDF syntax supporting reading/ writing Turtle and NTriples"
@@ -15,11 +15,7 @@
   ISC
 )
 makedepends=(
-  doxygen
-  mandoc
   meson
-  python-sphinx
-  python-sphinxygen
 )
 source=(https://download.drobilla.net/$pkgname-$pkgver.tar.xz{,.sig})
 sha512sums=('1ceb7caaa97195e829bec0436930c090698a6be8ba1b7ee0ef3f013c817dd7ae472bb9bcc271e55e8b6e12d110b2d1d3d8fe20c36672d6e63c53f151173cd897'
@@ -39,7 +35,7 @@
 }
 
 build() {
-  arch-meson $pkgname-$pkgver build
+  arch-meson $pkgname-$pkgver build -Ddocs=disabled -Dhtml=disabled -Dman=disabled -Dman_html=disabled -Dsinglehtml=disabled
   meson compile -C build
 }
 
@@ -49,25 +45,11 @@
 
 package_serd() {
   depends=(glibc)
-  optdepends=('serd-docs: for developer documentation')
   provides=(libserd-0.so)
 
   meson install -C build --destdir "$pkgdir"
   install -vDm 644 $pkgbase-$pkgver/LICENSES/* -t "$pkgdir/usr/share/licenses/$pkgname/"
 
-  (
-    cd "$pkgdir"
-    _pick $pkgname-docs usr/share/doc
-  )
-
   install -vDm 644 $pkgname-$pkgver/COPYING -t "$pkgdir/usr/share/licenses/$pkgname/"
   install -vDm 644 $pkgname-$pkgver/{AUTHORS,NEWS,README.md} -t "$pkgdir/usr/share/doc/$pkgname/"
 }
-
-package_serd-docs() {
-  pkgdesc+=" - documentation"
-
-  mv -v $pkgname/* "$pkgdir"
-  mv -v "$pkgdir/usr/share/doc/"serd{-0,}
-  install -vDm 644 $pkgbase-$pkgver/LICENSES/* -t "$pkgdir/usr/share/licenses/$pkgname/"
-}
