--- a/PKGBUILD
+++ b/PKGBUILD
@@ -121,30 +121,7 @@
   mv gtk4/* "$pkgdir"
 }
 
-package_libportal-qt5() {
-  pkgdesc+=" - Qt 5 backend"
-  depends+=(
-    "libportal=$pkgver-$pkgrel"
-    libstdc++
-    qt5-base
-    qt5-x11extras
-  )
-  provides=(libportal-qt5.so)
 
-  mv qt5/* "$pkgdir"
-}
-
-package_libportal-qt6() {
-  pkgdesc+=" - Qt 6 backend"
-  depends+=(
-    "libportal=$pkgver-$pkgrel"
-    libstdc++
-    qt6-base
-  )
-  provides=(libportal-qt6.so)
-
-  mv qt6/* "$pkgdir"
-}
 
 package_libportal-docs() {
   pkgdesc+=" - API documentation"
