--- a/PKGBUILD
+++ b/PKGBUILD
@@ -4,7 +4,6 @@
 pkgbase=gdk-pixbuf2
 pkgname=(
   gdk-pixbuf2
-  gdk-pixbuf2-docs
 )
 pkgver=2.44.7
 pkgrel=1
@@ -15,11 +14,12 @@
 depends=(
   glib2
   glibc
-  glycin
+  libjpeg-turbo
+  libpng
+  libtiff
   shared-mime-info
 )
 makedepends=(
-  gi-docgen
   git
   glib2-devel
   gobject-introspection
@@ -43,19 +43,19 @@
   local meson_options=(
     -D android=disabled
     -D builtin_loaders=all
-    -D documentation=true
-    -D gif=disabled
-    -D glycin=enabled
-    -D gtk_doc=true
+    -D documentation=false
+    -D gif=enabled
+    -D glycin=disabled
+    -D gtk_doc=false
     -D installed_tests=false
     -D introspection=enabled
-    -D jpeg=disabled
+    -D jpeg=enabled
     -D legacy_xpm=enabled
     -D man=true
     -D others=enabled
-    -D png=disabled
+    -D png=enabled
     -D thumbnailer=disabled
-    -D tiff=disabled
+    -D tiff=enabled
   )
 
   arch-meson gdk-pixbuf build "${meson_options[@]}"
@@ -72,17 +72,8 @@
 
   meson install -C build --destdir "$pkgdir"
 
-  mkdir -p doc/usr/share
-  mv {"$pkgdir",doc}/usr/share/doc
 
   install -Dt "$pkgdir/usr/share/libalpm/hooks" -m644 *.hook
 }
 
-package_gdk-pixbuf2-docs() {
-  pkgdesc+=" (documentation)"
-  depends=()
-
-  mv doc/* "$pkgdir"
-}
-
 # vim:set sw=2 sts=-1 et:
