--- a/PKGBUILD
+++ b/PKGBUILD
@@ -17,13 +17,12 @@
   GPL-2.0-or-later
   LGPL-2.0-or-later
 )
-_glibver=2.86.3
+_glibver=2.88.3
 makedepends=(
   "glib2=$_glibver"
   cairo
   git
   glibc
-  gtk-doc
   libffi
   libffi
   meson
@@ -31,7 +30,6 @@
   python-mako
   python-markdown
   python-setuptools
-  python-sphinx
 )
 source=(
   "git+https://gitlab.gnome.org/GNOME/gobject-introspection.git#tag=$pkgver"
@@ -39,7 +37,7 @@
   "git+https://gitlab.gnome.org/GNOME/gobject-introspection-tests.git"
 )
 b2sums=('46007e1c426ebfde0d94b84398a1d99e0fc516f1b5a78590d0d1ddec76b90d19a9c6f4f9959a6d243cbb3601390763bbdd7d05d1d1c3d4fb9746f453b40ea849'
-        'c384d8196e7966261d78245b59a0d14a061d49f7fb15faebd6e10527eb58c2893bffaee30f894b45feaeb1e1d8ccfc65e79958c42458b298e1e9675f68202780'
+        'SKIP'
         'SKIP')
 validpgpkeys=(
   923B7025EE03C1C59F42684CF0942E894B2EAFA0  # Philip Withnall <philip@tecnocode.co.uk>
@@ -57,7 +55,7 @@
 build() {
   local meson_options=(
     -D glib_src_dir="$srcdir/glib"
-    -D gtk_doc=true
+    -D gtk_doc=false
   )
 
   arch-meson $pkgbase build "${meson_options[@]}"
@@ -103,7 +101,6 @@
   _pick libg usr/lib/pkgconfig/gobject-introspection*-1.0.pc
   _pick libg usr/lib/girepository-1.0/GIRepository-2.0.typelib
   _pick libg usr/share/gir-1.0/GIRepository-2.0.gir
-  _pick libg usr/share/gtk-doc
 
   _pick runtime usr/lib/girepository-1.0
 }
