--- a/PKGBUILD
+++ b/PKGBUILD
@@ -3,7 +3,7 @@
 # Contributor: Joel Teichroeb <joel@teichroeb.net>
 
 pkgbase=wayland
-pkgname=(wayland wayland-docs)
+pkgname=(wayland)  # kashira: no docs split
 pkgver=1.26.0
 pkgrel=1
 pkgdesc='A computer display server protocol'
@@ -11,7 +11,7 @@
 url='https://wayland.freedesktop.org/'
 license=('MIT')
 depends=('glibc' 'libffi' 'expat' 'libxml2' 'default-cursors')
-makedepends=('meson' 'libxslt' 'doxygen' 'xmlto' 'graphviz' 'docbook-xsl' 'mdbook')
+makedepends=('meson' 'libxslt' 'xmlto' 'docbook-xsl')  # kashira: no doxygen/graphviz/mdbook docs
 validpgpkeys=('C7223EBE4EF66513B892598911A30156E0E67611'  # Bryce Harrington
               'C0066D7DB8E9AC6844D728715E54498E697F11D7'  # Derek Foreman
               '34FF9526CFEF0E97A340E2E40FDE7BE0E88F5E48') # emersion <contact@emersion.fr>
@@ -32,17 +32,8 @@
   provides=(libwayland-{client,cursor,egl,server}.so)
 
   meson install -C build --destdir "$pkgdir"
-  mkdir -p docs/share
-  mv "$pkgdir"/usr/share/{doc,man} docs/share
   install -Dm 644 $pkgbase-$pkgver/COPYING "$pkgdir/usr/share/licenses/$pkgname/COPYING"
 }
 
-package_wayland-docs() {
-  pkgdesc+=" (documentation)"
-  depends=()
-
-  mv docs "$pkgdir/usr"
-  install -Dm 644 $pkgbase-$pkgver/COPYING "$pkgdir/usr/share/licenses/$pkgname/COPYING"
-}
 
 # vim:set sw=2 sts=-1 et:
