--- a/PKGBUILD
+++ b/PKGBUILD
@@ -21,14 +21,8 @@ depends=(
   systemd-libs
 )
 makedepends=(
-  docbook-xml
-  docbook-xsl
-  docbook5-xml
   flex
-  fop
   git
-  libxslt
   meson
-  w3m
 )
 provides=(
@@ -66,5 +60,6 @@ prepare() {
 
 build() {
   arch-meson "${pkgname}" \
+    -Ddocs=disabled \
     -Dlogind=enabled \
     -Deconf=disabled \
@@ -85,9 +80,6 @@ package() {
   meson install -C build --destdir "${pkgdir}"
   install -Dm 644 $pkgname.tmpfiles "${pkgdir}"/usr/lib/tmpfiles.d/${pkgname}.conf
 
-  # remove unreproducible pdf files
-  rm "${pkgdir}"/usr/share/doc/Linux-PAM/*.pdf
-
   # set unix_chkpwd uid
   chmod +s "${pkgdir}"/usr/bin/unix_chkpwd
 }
