--- a/PKGBUILD
+++ b/PKGBUILD
@@ -74,7 +74,6 @@
   srt
   svt-av1
   v4l-utils
-  vapoursynth
   vid.stab
   vmaf
   vulkan-icd-loader
@@ -82,7 +81,6 @@
   x265
   xvidcore
   xz
-  zeromq
   zimg
   zlib
 )
@@ -198,13 +196,11 @@
     --enable-libxml2 \
     --enable-libxvid \
     --enable-libzimg \
-    --enable-libzmq \
     --enable-nvdec \
     --enable-nvenc \
     --enable-opencl \
     --enable-opengl \
     --enable-shared \
-    --enable-vapoursynth \
     --enable-version3 \
     --enable-vulkan \
     --disable-decoder=magicyuv # pixelsmash
@@ -238,7 +234,6 @@
 #   libx265.so
     libxvidcore.so
     libzimg.so
-    libzmq.so
   )
 
   make DESTDIR="${pkgdir}" -C ffmpeg install install-man
