--- a/PKGBUILD
+++ b/PKGBUILD
@@ -10,8 +10,7 @@
 arch=(x86_64)
 license=(MIT)
 depends=(glibc libglvnd libx11)
-makedepends=(git meson mesa-libgl systemd doxygen graphviz)
-checkdepends=(xorg-server-xvfb)
+makedepends=(git meson systemd)
 _commit=c84bc9459357a40e46e2fec0408d04fbdde2c973  # tags/1.5.10^0
 source=("git+https://github.com/anholt/libepoxy#commit=$_commit")
 sha256sums=('SKIP')
@@ -31,11 +30,6 @@
   meson compile -C build
 }
 
-check() {
-  xvfb-run -s '-nolisten local' \
-    meson test -C build --print-errorlogs
-}
-
 package_libepoxy() {
   provides=(libepoxy.so)
 
