--- a/PKGBUILD
+++ b/PKGBUILD
@@ -8,7 +8,6 @@
   harfbuzz-cairo
   harfbuzz-icu
   harfbuzz-utils
-  harfbuzz-docs
 )
 pkgver=14.5.0
 pkgrel=1
@@ -18,66 +17,49 @@
 license=(MIT)
 makedepends=(
   cairo
-  chafa
   freetype2
-  git
-  glew
-  glfw
+  gcc
   glib2
   glib2-devel
   glibc
   gobject-introspection
   graphite
-  gtk-doc
   help2man
   icu
-  libgl
   libpng
   libstdc++
   meson
   python
-  ragel
   zlib
 )
-checkdepends=(
-  python-fonttools
-  python-setuptools
-)
 source=(
-  "git+https://github.com/harfbuzz/harfbuzz?signed#tag=$pkgver"
-)
-b2sums=('275cea3ab5c5ee27a114d60b08c386c8552b29c2014a85606696324ad3f3a8e4f087e363e9e67af7456a8fef2cb8cbde2da81b18615be42e4166ae4400a6b025')
-validpgpkeys=(
-  053D20F17CCCA9651B2C6FCB9AB24930C0B997A2 # Khaled Hosny <khaled@aliftype.com> (@khaledhosny)
-  9F377DDB6D3153A48EB3EB1E63CC496475267693 # Caleb Maclennan <caleb@alerque.com> (@alerque)
-  2277650A4E8BDFE4B7F6BE419FEE04E5D3531115 # Ebrahim Byagowi <ebrahim@gnu.org> (@ebraminio)
-  EACF64F53455E2771BA661A4803B21859F015E4E # Behdad Esfahbod <behdad@behdad.org> (@behdad)
+  "https://github.com/harfbuzz/harfbuzz/releases/download/$pkgver/harfbuzz-$pkgver.tar.xz"
 )
+b2sums=('cde386fc963c56c3e2d352cb66419094bad64348752072a94856e3229d0910d127c5922bbc980b8a663ce138529e7c2ac2a6715c5fa468057148c126e40e4164')
 
 prepare() {
-  cd harfbuzz
+  cd harfbuzz-$pkgver
 }
 
 build() {
   local meson_options=(
     -D cpp_std=c++17
     -D graphite2=enabled
+    -D chafa=disabled
+    -D docs=disabled
+    -D gpu=disabled
+    -D gpu_demo=disabled
+    -D tests=disabled
   )
 
   # Harfbuzz wants no exceptions
   CFLAGS="${CFLAGS/-fexceptions/}"
   CXXFLAGS="${CXXFLAGS/-fexceptions/}"
 
-  arch-meson harfbuzz build "${meson_options[@]}"
+  arch-meson harfbuzz-$pkgver build "${meson_options[@]}"
   meson compile -C build
 }
 
-check() {
-  mkdir -p tmp
-  TMPDIR="$PWD/tmp" meson test -C build --print-errorlogs --no-rebuild
-  rm -r tmp
-}
-
 _pick() {
   local p="$1" f d; shift
   for f; do
@@ -113,12 +95,10 @@
     _pick hb-icu usr/include/harfbuzz/hb-icu.h
 
     _pick hb-utils usr/bin
-    _pick hb-utils usr/share/man/man1
 
-    _pick hb-docs usr/share/gtk-doc
   )
 
-  install -Dt "$pkgdir/usr/share/licenses/$pkgname" -m644 harfbuzz/COPYING
+  install -Dt "$pkgdir/usr/share/licenses/$pkgname" -m644 harfbuzz-$pkgver/COPYING
 }
 
 package_harfbuzz-cairo() {
@@ -135,7 +115,7 @@
 
   mv hb-cairo/* "$pkgdir"
 
-  install -Dt "$pkgdir/usr/share/licenses/$pkgname" -m644 harfbuzz/COPYING
+  install -Dt "$pkgdir/usr/share/licenses/$pkgname" -m644 harfbuzz-$pkgver/COPYING
 }
 
 package_harfbuzz-icu() {
@@ -149,36 +129,23 @@
 
   mv hb-icu/* "$pkgdir"
 
-  install -Dt "$pkgdir/usr/share/licenses/$pkgname" -m644 harfbuzz/COPYING
+  install -Dt "$pkgdir/usr/share/licenses/$pkgname" -m644 harfbuzz-$pkgver/COPYING
 }
 
 package_harfbuzz-utils() {
   pkgdesc+=" - Utilities"
   depends=(
     cairo libcairo.so
-    chafa libchafa.so
     freetype2 libfreetype.so
-    glew libGLEW.so
-    glfw
     glib2 libg{lib,object}-2.0.so
     glibc
     harfbuzz libharfbuzz{,-gobject,-subset}.so
     harfbuzz-cairo libharfbuzz-cairo.so
-    libgl libGL.so
-    libstdc++ libstdc++.so
   )
 
   mv hb-utils/* "$pkgdir"
 
-  install -Dt "$pkgdir/usr/share/licenses/$pkgname" -m644 harfbuzz/COPYING
-}
-
-package_harfbuzz-docs() {
-  pkgdesc+=" - Documentation"
-
-  mv hb-docs/* "$pkgdir"
-
-  install -Dt "$pkgdir/usr/share/licenses/$pkgname" -m644 harfbuzz/COPYING
+  install -Dt "$pkgdir/usr/share/licenses/$pkgname" -m644 harfbuzz-$pkgver/COPYING
 }
 
 # vim:set sw=2 sts=-1 et:
