--- a/PKGBUILD
+++ b/PKGBUILD
@@ -14,7 +14,7 @@
 source=(https://samba.org/ftp/tevent/${pkgname}-${pkgver}.tar{.gz,.asc})
 license=('GPL-3.0-or-later')
 depends=('talloc')
-makedepends=('python' 'cmocka')
+makedepends=('python')
 optdepends=('python: for python bindings')
 provides=(libtevent.so)
 validpgpkeys=('9147A339719518EE9011BCB54793916113084025') # Samba Library Distribution Key <samba-bugs@samba.org>
