--- a/PKGBUILD
+++ b/PKGBUILD
@@ -16,10 +16,8 @@
 license=(MIT)
 makedepends=(
   bash-completion
-  doxygen
   git
   glibc
-  graphviz
   libxcb
   libxml2
   meson
@@ -46,7 +44,7 @@
     -D enable-docs=true
   )
 
-  arch-meson libxkbcommon build "${meson_options[@]}"
+  arch-meson libxkbcommon build -D enable-docs=false  # kashira: no doxygen "${meson_options[@]}"
   meson compile -C build
 }
 
