--- a/PKGBUILD
+++ b/PKGBUILD
@@ -28,16 +28,15 @@
          systemd-libs
          zstd)
 makedepends=(bash-completion
-             gi-docgen
-             git
-             glib2-devel
-             gobject-introspection
-             gperf
-             itstool
-             meson
-             qt6-tools
-             vala
-             xmlto)
+ gi-docgen
+ git
+ glib2-devel
+ gobject-introspection
+ gperf
+ itstool
+ meson
+ vala
+ xmlto)
 source=(git+https://github.com/ximion/appstream#tag=v$pkgver?signed
         update-appstream-cache.hook)
 sha256sums=('8699c33219c1cb4aa82c57cd9777f9a34cdd14ac6558c93a3553420c725f4a24'
